chain/indexer/src/streamer/streamer.rs

```diff
@@ -31,24 +32,31 @@ impl From<MailboxError> for FailedToFetchData {
     }
 }
 
-struct FetchBlockResponse {
-    block_response: StreamerMessage,
-    new_outcomes_to_get: Vec<types::TransactionOrReceiptId>,
-}
-
 /// Resulting struct represents block with chunks
 #[derive(Debug)]
 pub struct StreamerMessage {
     pub block: views::BlockView,
-    pub chunks: Vec<views::ChunkView>,
-    pub outcomes: Vec<Outcome>,
+    pub chunks: Vec<IndexerChunkView>,
+    pub outcomes: HashMap<CryptoHash, views::ExecutionOutcomeWithIdView>,
```

Comment:
Let's rename it to `receipt_execution_outcomes` to be immediately obvious to the user.